### DISCUSSION TITLE
Can't get spotify to work

### DISCUSSION THREAD

**Discussion by AaryaHC** [punxaphil/custom-sonos-card#428] [Q&A]
I can't get Spotify to work through the 'Browse Media' pop up menu. I keep getting the notification 'Media Browsing Error: Invalid Spotify account specified'. Spotify is integrated into HA fine however, and I can control and browse Spotify within the HA Media tab no problem. Any assistance would be appreciated.

**Comment by punxaphil** [comment]
This is surprising. Not sure what the solution could be 🤔
